smsdk/client.py

```diff
@@ -702,17 +805,21 @@ def get_machine_type_names(self, clean_strings_out=True):
         else:
             return machine_types["source_type"].unique().tolist()
 
-    def get_raw_data(
-        self,
-        raw_data_table,
-        normalize=True,
-        fields=[],
-        time_selection=one_day_relative,
-        limit=400,
-        offset=0,
-        *args,
-        **kwargs,
-    ):
+    def get_raw_data(self, *args, normalize=True, **kwargs):
```

Comment:
= None the args

Reply:
Not required to explicitly assigning None to args.